CMakeLists.txt

```diff
@@ -76,6 +76,7 @@ list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake/module)
 #=============================
 include(CMakeDependentOption)
 # When adding a new option, end the <help_text> with a full stop for consistency.
+option(BUILD_BITCOIN_BIN "Build bitcoin executable." ON)
```

Comment:
Can you add this to `Executables:` in the summary as well?

Reply:
> Can you add this to `Executables:` in the summary as well?

Good idea, added now

Reply:
Thanks, that worked.